ci/scripts/benchmark.sh

```diff
@@ -45,38 +59,63 @@ function cleanup {
 
 trap cleanup EXIT
 
+DB_USER=dbuser
+DB_PWD=dbpwd
+
 if [[ -z "${RISINGWAVE_IMAGE_TAG+x}" ]]; then
   IMAGE_TAG="latest"
 else
   IMAGE_TAG="${RISINGWAVE_IMAGE_TAG}"
 fi
 
+if [ -z "${BENCH_SKU+x}" ] || [ "${BENCH_SKU}" == "MultiNodeBench" ]; then
+  SKU="multinode"
```

Comment:
What is the purpose of SKU? can it have only 2 possible values?

Reply:
The SKU is provided by the cloud to create risingwave specifications. Currently, the cloud provides three types of sku: FreeTier, SingleNodeBench, and MultiNodeBench. For bench testing, we only need to create two risingwaves：SingleNodeBench and MultiNodeBench.